draft-ietf-tls-ctls.md

```diff
@@ -374,15 +223,40 @@ group is listed by the code point name in {{RFC8446}}, Section 4.2.7.
 (e.g., ed25519). This implies a literal "signature_algorithms" extension
 consisting solely of this group.
 
-randomSize (integer):
+random (integer):
 : indicates that the ClientHello.Random and ServerHello.Random values
-are truncated to the given values. When the transcript is
+are truncated to the given length. When the transcript is
 reconstructed, the Random is padded to the right with 0s and the
 anti-downgrade mechanism in {{RFC8446)}, Section 4.1.3 is disabled.
 IMPORTANT: Using short Random values can lead to potential
-attacks. When Random values are shorter than 8 bytes, PSK-only modes
-MUST NOT be used, and each side MUST use fresh DH ephemerals.
-The Random length MUST be less than or equal to 32 bytes.
+attacks. The Random length MUST be less than or equal to 32 bytes.
+
+mutualAuth (boolean):
+: indicates whether the client has to be prepared to transmit a 
+Certificate and a CertificateVerify message and whether the server 
+has to transmit a CertificateRequest message. 
+
+extension_order: 
+: indicates in what order extensions appear in respective messages. 
+This allows to omit sending the type. If there is only a single 
+extension to be transmitted, then the extension length field can also
+be omitted. For example, imagine that only the KeyShare extension 
```

Comment:
I think we'd have to explain how we knew that to be the case. There's no indication that this is the exclusive list.

Also, aren't all the extensions self-delimiting? If so, can we always remove the extension length?

At this point do you just want to put an OPEN ISSUE here?